flask/app/models.py

```diff
@@ -421,25 +424,31 @@ class Variation(str, Enum):
     Stop_Lost = "Stop_Lost"
     Start_lost = "Start_lost"
     Protein_Altering_Variant = "Protein_Altering_Variant"
-    Start_Retrained_Variant = "Start_Retrained_Variant"
+    Start_Retained_Variant = "Start_Retained_Variant"
     Mature_miRNA_Variant = "Mature_miRNA_Variant"
-    Stop_Retained_Variant = "Stop_Retained_Variant"
 
 
+@dataclass
 class Gene(db.Model):
     gene_id: int = db.Column(db.Integer, primary_key=True)
-    hgnc_gene_id: int = db.Column(db.Integer)
+    hgnc_gene_id: int = db.Column(db.Integer, unique=True)
     ensembl_id: int = db.Column(db.Integer)
+    gene: str = db.Column(db.String(50))
     hgnc_gene_name: str = db.Column(db.String(50))
+    variant = db.relationship("Variant", back_populates="gene")
```

Comment:
How is this relationship supposed work?

Reply:
it is suppose to be a one-to-one relationship between gene_id and variant_id but it looks like i'm missing the `useList = False` argument making it currently a one-to-many